eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -235,7 +235,7 @@ stages:
         - ${{ each artifact in parameters.Artifacts }}:
           - ${{if ne(artifact.skipPublishDevFeed, 'true')}}:
             - pwsh: |
-                $detectedPackageName=Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.tgz
+                $detectedPackageName=Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*-alpha.*.tgz
```

Comment:
Can you give more information of how you got into this job without a alpha version? 

Also what does this step do if $detectedPackageName isn't set? Do we need to add extra handling below?